client/src/Components/Home/Home.js

```diff
@@ -1,17 +1,22 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import './Home.css';
 
 const Home = () => {
+  const [blurb, setBlurb] = useState(false);
 
   return (
     <section className='home'>
       <h1 className='logo-home'>BEAM</h1>
-      <h2>Beam is an interactive self-care feeling tracking app.</h2>
-        <h3 className='blurb'>
-        We created a flow intended to help you work through any kind of feeling or emotion you might be experiencing. You'll be asked to choose a feeling and reflect on it. Add anything about your experience that might help you avoid or recreate the feeling in the future (depending on the feeling). After your reflection, you'll see an offering of a few activities that might help. <br></br>Your entries will be available to you in the Feelings Archive for future reflection. 
-        </h3> 
-      <h2>We hope this process will help reveal useful patterns over time.</h2>
+      <p className='uni-btn' onClick={() => setBlurb(!blurb)}>What is Beam?</p>
+      <div>
+        {blurb 
+          ? <div className='blurb-section'>
+              <h2>Beam is an interactive self-care feeling tracking app.</h2>
+              <h3 className='blurb'>We created a flow intended to help you work through any kind of feeling or emotion you might be experiencing. You'll be asked to choose a feeling and reflect on it. Add anything about your experience that might help you avoid or recreate the feeling in the future (depending on the feeling). After your reflection, you'll see an offering of a few activities that might help. <br/><br/>Your entries will be available to you in the Feelings Archive for future reflection. <br/> <br/>We hope this process will help reveal useful patterns over time.</h3> 
+            </div> 
+          : null}
```

Comment:
this might be a place where you can use the `&&` logic, if blurb is true, then return the div